docs/install-w-machine.md

```diff
@@ -101,20 +101,20 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
 
         $ docker-machine ls
         NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER   ERRORS
-        agent1    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.9.1   
-        agent2    -        virtualbox   Running   tcp://192.168.99.103:2376           v1.9.1   
-        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1   
+        agent1    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.9.1
+        agent2    -        virtualbox   Running   tcp://192.168.99.103:2376           v1.9.1
+        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1
 
 
 2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary Swarm manager on `manager`.
 
-        $ docker run -d -p <your_selected_port>:2375 -t swarm manage token://<cluster_id> >
+        $ docker run -d -p <your_selected_port>:3376 -t -v /var/lib/boot2docker:/certs:ro swarm manage -H 0.0.0.0:3376 --tlsverify --tlscacert=/certs/ca.pem --tlscert=/certs/server.pem --tlskey=/certs/server-key.pm token://<cluster_id>
```

Comment:
I made a fix in #1784. Sorry about the complication, hope it's possible to pull that commit in too!